index.rst

```diff
@@ -372,30 +372,41 @@ c.l.o stubb
 
   Here is where we currently are in the release process. Current step in bold.
 
-  Precursor Steps
+  Release Precursor Steps
   ---------------------------------
 
   1. Identify any pre-release blockers ("must-have features") :tools:
-  2. Wait for them to clear
+  Contributors check if there are outstanding issues that have to be included in the next M.m.p release and related them as blocker to the above issue DM-XXXXX.
+  1. Create Jira issues for each release activity.
+
+  Release Jira issue: https://jira.lsstcorp.org/browse/DM-XXXXX
+  Tentative weekly to use as starting point for the release is w_20YY_WW
+  Tentative target date to close the release is YYYY-MM-DD.
 
   Release Engineering Steps
   -------------------------------
 
-  1. Branch 888.0.x of newinstall.sh
-  1. Build and publish rc1 release candidate (based on w.9999.52)
-  1. **Wait for first round of bugs to clear**
-  1. Build and publish additional rcX releases if/as necessary
-  1. Full OS testing (see https://ls.st/faq )
-  1. Build and publish final release
+  1. Check that the weekly build is scientifically suitable to be used as starting point for the release
+  1. Create first release candidate vM.m.p.rc1
+  1. Release candidate vM.m.p.rc1 available:
+   - Build: bxxxxx
+   - Weekly: w_20YY_WW
+  1. Build the release candidate on supported platforms.
```

Comment:
Isn't there at least some level of testing during the build process?
```suggestion
  1. Build and test the release candidate on supported platforms.
```
Previously the process said that bugs were fixed before building on supported platforms; is this now done for every RC?

Reply:
The demo has been removed officially via RFC discussed at the CCB a few weeks ago. Therefore, until there is a different test I can do, I can only build.

Reply:
Don't the builds do at least some minimal (unit) testing?

Reply:
Yes, but unit tests are usually part of the build process. If we add here a reference to a test activity, it has to be something more than unit tests.